roll-network/tryrolljs
# Enhancement: Modify Batch Send example to allow the user to send multiple batches of transactions with different parameters and display the transaction details

## What's done
- Modified the example of sendBatchFromPlatformUser so that the user is prompted to provide transaction parameters multiple times within a while loop, allowing them to send multiple batches of transactions.
- The modified version introduces an array called transactions to store individual transaction parameters for each batch.
- After each transaction is completed, the transaction details are stored in the transactionDetails object and then passed to the printTable function to display the transaction details in a table.
- The while loop continues as long as the user chooses to send another batch by setting batchSendPrompt to answers.batchSendAgain.

## How to test

Verify the example by choosing Send Batch From Platform User in the command line prompt.

## Demo (screenshots, recordings, etc.)
<img width="887" alt="Screenshot 2023-06-22 at 10 45 45" src="https://github.com/roll-network/tryrolljs/assets/98424384/654ddcdb-6d2b-4b1c-a653-eba895eb5a93">
